test/OpenTelemetry.Tests/Metrics/MultipleReadersTests.cs

```diff
@@ -100,6 +141,17 @@ public void SdkSupportsMultipleReaders(AggregationTemporality aggregationTempora
             // Check value exported for Gauge
             this.AssertLongSumValueForMetric(exportedItems1[1], 300);
             this.AssertLongSumValueForMetric(exportedItems2[1], 400);
+
+            // Check value exported for ObservableCounter
+            this.AssertLongSumValueForMetric(exportedItems1[2], 300);
```

Comment:
I think it may not be very obvious to the future readers on this code.. I'll send some refactoring of these tests to make it more intuitive, in a future PR.